client/src/frontend/components/resource/Card/styles.js

```diff
@@ -0,0 +1,105 @@
+import styled from "@emotion/styled";
+import IconComposer from "global/components/utility/IconComposer";
+import {
+  respond,
+  defaultTransitionProps,
+  subtitlePrimary
+} from "theme/styles/mixins";
+
+export const Card = styled.article`
+  --Thumbnail-Icon-background-color: transparent;
+
+  display: flex;
+`;
+
+export const Preview = styled.div`
+  display: flex;
+  width: 37.8%;
+
+  ${respond(`width: 155px;`, 40)}
+
+  ${respond(`width: 135px;`, 75)}
+
+  ${respond(`width: 155px;`, 85)}
+`;
+
+export const InfoLink = styled.a`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 62.2%;
+  padding: 12px 15px;
+  text-decoration: none;
+  background-color: var(--color-base-neutral05);
+
+  ${respond(`width: calc(100% - 155px);`, 40)}
+
+  ${respond(`width: calc(100% - 135px);`, 40)}
+
+  ${respond(`width: calc(100% - 155px);`, 40)}
```

Comment:
These are all set at the same breakpoint! 🤔

Reply:
Ha! Did I do that?